# patch to enable linux-x86 arch in main GAMESS compile driver

--- gamess/compall	2005-07-18 22:37:23.000000000 +0000
+++ gamess-gentoo/compall	2005-09-04 15:00:18.000000000 +0000
@@ -12,8 +12,8 @@
 #      linux-ia64,linux-pc,mac32,macG5,necsx,sgi32,sgi64,sun32,sun64
 #  2. Choose your GAMESS directory tree.
 #
-set TARGET=ibm64
-chdir /u1/mike/gamess
+set TARGET=linux-pc
+#chdir /u1/mike/gamess
 #
 #  ----- and now the script begins...
 #
